app/models/audit/event.rb

```diff
@@ -10,9 +10,23 @@ class Event < Util::Struct
     progname 'conjur'
 
     def log_to logger
+      log_to_db
       logger.log logger_severity, self, progname
     end
 
+    def log_to_db
```

Comment:
What about passing these arguments into the `log_to_db` method?  It would make unit testing this a snap.